packages/config/config/devices/0x027a/zen16.json

```diff
@@ -0,0 +1,603 @@
+					"value": 4
+				}
+			]
+		},
+		"2": {
+			"label": "Switch Type for Relay 1 (Sw1)",
+			"description": "Choose the wall switch type you want to connect to the Sw1 terminal.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 2,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Momentary Switch",
+					"value": 0
+				},
+				{
+					"label": "Toggle switch (follow switch)",
+					"value": 1
+				},
+				{
+					"label": "Toggle switch (change state on toggle)",
+					"value": 2
+				},
+				{
+					"label": "Garage door (momentary mMode for Z-Wave control)",
+					"value": 3
+				}
+			]
+		},
+		"3": {
+			"label": "Switch Type for Relay 2 (Sw2)",
+			"description": "Choose the wall switch type you want to connect to the Sw2 terminal.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 2,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Momentary Switch",
+					"value": 0
+				},
+				{
+					"label": "Toggle switch (follow switch)",
+					"value": 1
+				},
+				{
+					"label": "Toggle switch (change state on toggle)",
+					"value": 2
+				},
+				{
+					"label": "Garage door (momentary mMode for Z-Wave control)",
+					"value": 3
+				}
+			]
+		},
+		"4": {
+			"label": "Switch Type for Relay 3 (Sw3)",
+			"description": "Choose the wall switch type you want to connect to the Sw3 terminal.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 2,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Momentary Switch",
+					"value": 0
+				},
+				{
+					"label": "Toggle switch (follow switch)",
+					"value": 1
+				},
+				{
+					"label": "Toggle switch (change state on toggle)",
+					"value": 2
+				},
+				{
+					"label": "Garage door (momentary mMode for Z-Wave control)",
+					"value": 3
+				}
+			]
+		},
+		"5": {
+			"label": "LED Indicator Control",
+			"description": "Choose the LED indicator behavior",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "On when all relays are off",
+					"value": 0
+				},
+				{
+					"label": "On when any relays is on",
+					"value": 1
+				},
+				{
+					"label": "Always off",
+					"value": 2
+				},
+				{
+					"label": "Always on",
+					"value": 3
+				}
+			]
+		},
+		"6": {
+			"label": "Auto Turn-Off Timer for Relay 1",
+			"description": "Sets the time after which you want relay 1 to automatically turn off once it has been turned on.",
+			"valueSize": 4,
+			"unit": "minutes (unless changd by parameter 15)",
```

Comment:
That's a bit long for a unit. If we really cannot say for sure, it's probably better to not have a unit here. (Best leave a comment)

Reply:
That’s from the manual. 

Auto Turn-Off Timer

Parameter 6: Use this parameter to set the time after which you want Relay 1 to automatically turn off once it has been turned on. The number entered as value corresponds to the number of minutes, seconds, or hours, depending on the value entered in Parameter 15.
Values: 0 – timer disabled (default); 1 – 65535.
Size: 4 byte dec

Reply:
I changed the description to reference the parameter. I did not include a comment (yet) because those get stripped out when I process files and I still need to process the zwave ids into these.